ledger-tool/src/main.rs

```diff
@@ -873,7 +881,7 @@ fn main() {
             let starting_slot = value_t_or_exit!(arg_matches, "starting_slot", Slot);
             let allow_dead_slots = arg_matches.is_present("allow_dead_slots");
             output_ledger(
-                open_blockstore(&ledger_path),
+                open_blockstore(&ledger_path, AccessType::AllowSecondary),
```

Comment:
TODO: carefully audit for the appropriate AccessType for all of subcommands.

Reply:
Done, I've tested all the subcommands and they worked as intended.

@mvines FYI, I found `graph` to be very useful to create a graph right from a still running validator's ledger. :)

Reply:
totally, this is gonna be yuuuuge!